dataGenerator/s3Images.js

```diff
@@ -0,0 +1,32 @@
+const AWS = require('aws-sdk');
+AWS.config.loadFromPath('s3Config.json');
+const s3 = new AWS.S3();
+
+const params = {
+  Bucket: 'sdc-andrew-movie-posters',
+  MaxKeys: 800,
+};
+
+const listAllObjects = async () => {
+  const imgUrlArray = [];
+  try {
+    const data = await s3.listObjectsV2(params).promise();
+    data.Contents.forEach((elem) => {
```

Comment:
i think this is better as `imgUrlArray = data.Contents.map(...)`

(bc `forEach` + `push` === `map`)

Reply:
Will do.